### ISSUE TITLE
New web page for listening news

### ISSUE THREAD

**Issue by vidit624** [rsrkpatwari1234/EchoForMe#2]

**Comment by Janhabi-Mukherjee** [comment]
I know html/css....so can I work on this??.....or I need to know some other tech stack??

**Comment by rsrkpatwari1234** [comment]
Yeah @Janhabi-Mukherjee you may require a little bit idea of using backend in html pages as well. But it is very easy. You can take a look at other html pages and understand how it is working. But that work is only 30%, rest is all html and css.

Go ahead and take it up !! It will be a new learning experiencee for you all together

**Comment by Janhabi-Mukherjee** [comment]
thanks @rsrkpatwari1234 .........I will love to work on it under your guidance................if you can suggest me some resources to learn the backend part.

**Comment by rsrkpatwari1234** [comment]
@Janhabi-Mukherjee since this repo focuses on django, try following the links attached in the readme

**Comment by Janhabi-Mukherjee** [comment]
@rsrkpatwari1234 ok I am looking into it

**Comment by jatinjain001** [comment]
I would like to work on it , Please add the label DWOC and level tag on it .
